src/rgw/rgw_http_client.cc

```diff
@@ -121,7 +121,7 @@ int RGWHTTPClient::process(const char *method, const char *url)
 
   CURLcode status = curl_easy_perform(curl_handle);
   if (status) {
-    dout(0) << "curl_easy_perform returned error: " << error_buf << dendl;
+    dout(0) << "curl_easy_perform returned status " << status << " error: " << error_buf << dendl;
```

Comment:
which CURLcode are you seeing for these failures? can we map that code to `ERR_SERVICE_UNAVAILABLE` here, instead of going through `EINVAL`?